domains/enrique.json

```
@@ -0,0 +1,16 @@
+{
+  "owner": {
+    "username": "tecnocrata",
+    "email": "enrique@ortuno.net"
+  },
+  "records": {
+    "CNAME": "tecnocrata.vercel.app"
```

you cant use redirect_config unless its URL records (whic redirect and not replace your domain name)
```suggestion
    "URl": "https://tecnocrata.vercel.app"
```